src/main/java/com/plaid/client/request/InstitutionsSearchRequest.java

```diff
@@ -19,13 +19,20 @@
 public final class InstitutionsSearchRequest extends BasePublicRequest {
   private String query;
   private RequiredField<List<Product>> products = RequiredField.empty();
-
+  private Options options;
 
   public InstitutionsSearchRequest(String query) {
     notNull(query, "query");
     this.query = query;
   }
 
+  public InstitutionsSearchRequest(String query, boolean includeInstitutionData) {
+    notNull(query, "query");
+    this.query = query;
+    this.options = new Options(includeInstitutionData);
+  }
+
+
   public InstitutionsSearchRequest withProducts(Product... products) {
```

Comment:
can we add a .withIncludeInstitutionData(boolean) instead of creating more constructors?

Reply:
yes will do

Reply:
Uh. I'm not a big fan of object mutation here. Can we either make it a static factory method (ie. static method that always returns a new object)?